LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -95,16 +104,9 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-loan-typ-mtg;InvestmentHomeLoan">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-mtg;MortgageLoan"/>
+		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;Mortgage"/>
 		<rdfs:label xml:lang="en">investment home loan</rdfs:label>
-		<skos:definition xml:lang="en">A loan for a second or subsequent property which is taken out for investment purposes.</skos:definition>
-		<skos:editorialNote xml:lang="en">[formal definition to come from participants] The purpose for this is prospecitve income rather than personal use</skos:editorialNote>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;InvestmentHomeLoanContract">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanContract"/>
-		<rdfs:label xml:lang="en">investment home loan contract</rdfs:label>
-		<owl:disjointWith rdf:resource="&fibo-loan-typ-mtg;SecondHomeLoanContract"/>
+		<owl:disjointWith rdf:resource="&fibo-loan-typ-mtg;SecondHomeLoan"/>
```

Comment:
The disjointness does not make sense when the definition includes "second property". Is it not possible for a property to be purchased both for vacation use and as an investment?